src/gmt_io.c

```diff
@@ -753,8 +753,8 @@ GMT_LOCAL void gmtio_ogr_check_if_geographic (struct GMT_CTRL *GMT) {
 			return;
 		}
 	}
-	if (S->proj[GMTIO_GCS]) {	/* See if we have degree units */
-		if (strstr (S->proj[GMTIO_GCS], "UNIT[\"Degree\",0.0174532925199433]")) {
+	if (S->proj[GMTIO_WKT]) {	/* See if we have degree units */
+		if (strstr (S->proj[GMTIO_WKT], "UNIT[\"Degree\",0.0174532925199433]")) {
```

Comment:
No. This is wrong. Se for example
```
julia> epsg2proj(2000)
"+proj=tmerc +lat_0=0 +lon_0=-62 +k=0.9995 +x_0=400000 +y_0=0 +a=6378249.145 +rf=293.465 +units=m +no_defs"


julia> println(epsg2wkt(2000))
PROJCS["Anguilla 1957 / British West Indies Grid",
    GEOGCS["Anguilla 1957",
        DATUM["Anguilla_1957",
            SPHEROID["Clarke 1880 (RGS)",6378249.145,293.465,
                AUTHORITY["EPSG","7012"]],
            AUTHORITY["EPSG","6600"]],
        PRIMEM["Greenwich",0,
            AUTHORITY["EPSG","8901"]],
        UNIT["degree",0.0174532925199433,
```

Reply:
OK, I have removed that check.  So currently no check on WKT but presumably any of the other 2-3 will be more helpful.